snuba/settings/__init__.py

```diff
@@ -71,6 +71,19 @@
         },
         "single_node": True,
     },
+    {
+        "host": os.environ.get("CLICKHOUSE_HOST", "localhost"),
+        "port": int(os.environ.get("CLICKHOUSE_PORT", 9000)),
+        "user": os.environ.get("CLICKHOUSE_USER", "default"),
+        "password": os.environ.get("CLICKHOUSE_PASSWORD", ""),
+        "database": os.environ.get("CLICKHOUSE_DATABASE", "slice_1_default"),
+        "http_port": int(os.environ.get("CLICKHOUSE_HTTP_PORT", 8123)),
+        "storage_sets": {
+            "#slice(migrations,1)",
+            "#slice(generic_metrics_sets,1)",
+        },
+        "single_node": True,
+    },
```

Comment:
What is the rationale for adding a db for another slice in the local config? 
Is it only for your tests or do you expect to merge this part ?

Reply:
I just did this so that I could run `snuba migrate` locally. I think before merge I would add a second slice to `settings_test` so we could automate that migrations run correctly on a second slice, but this is going to be removed.